wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -771,14 +792,16 @@ Make sure to carefully read the section about [Pagination](#section/Pagination)
 leverage the API capabilities.
 |]
   where
-    createAccount    = decodeUtf8 $ encodePretty $ genExample @(WalletResponse Account)
-    createAddress    = decodeUtf8 $ encodePretty $ genExample @(WalletResponse WalletAddress)
-    createWallet     = decodeUtf8 $ encodePretty $ genExample @(WalletResponse Wallet)
-    readAccounts     = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Account])
-    readAddresses    = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Address])
-    readFees         = decodeUtf8 $ encodePretty $ genExample @(WalletResponse EstimatedFees)
-    readNodeInfo     = decodeUtf8 $ encodePretty $ genExample @(WalletResponse NodeInfo)
-    readTransactions = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Transaction])
+    createAccount        = decodeUtf8 $ encodePretty $ genExample @(WalletResponse Account)
+    createAddress        = decodeUtf8 $ encodePretty $ genExample @(WalletResponse WalletAddress)
+    createWallet         = decodeUtf8 $ encodePretty $ genExample @(WalletResponse Wallet)
+    readAccounts         = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Account])
+    readAccountBalance   = decodeUtf8 $ encodePretty $ genExample @(WalletResponse AccountBalance)
+    readAccountAddresses = decodeUtf8 $ encodePretty $ genExample @(WalletResponse AccountAddresses)
+    readAddresses        = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Address])
+    readFees             = decodeUtf8 $ encodePretty $ genExample @(WalletResponse EstimatedFees)
+    readNodeInfo         = decodeUtf8 $ encodePretty $ genExample @(WalletResponse NodeInfo)
+    readTransactions     = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Transaction])
```

Comment:
rather than large whitespace changes like this i would prefer an indentation to make the start point equal for each line. minor quibble :)

Reply:
And I do agree with you!